modules/rosapps/applications/devutils/createspec/CMakeLists.txt

```diff
@@ -1,4 +1,8 @@
 
+if(GCC AND NOT GCC_VERSION VERSION_LESS 7)
```

Comment:
Can we change this to `if(GCC AND GCC_VERSION VERSION_GREATER 7)`, which we already use in many places? (yes, it's not exactly the same, but we will never have GCC 7.x users anyway)
Easier to parse in the head in my opinion :)